spec/controllers/users_controller_spec.rb

```diff
@@ -0,0 +1,80 @@
+require 'rails_helper'
+
+RSpec.describe "UsersController" do
+include AuthenticationSpecHelper
+  before do
+    @controller = UsersController.new
+  end
+  describe 'GET #new' do   
+    context 'Render template new' do
+      it 'renders the `new` template' do
+        get :new
+        expect(response).to render_template('new')
+      end
+    end
+  end
+  describe 'POST #create' do  
+    let!(:user) { create(:user) }
+    context 'create with valid user' do
+      it 'create a new user' do
+        params = {
+          name: user.name,
+          email: 'testuser@gmail.com',
+          password: user.password_digest,
+          password_confirmation: user.password_digest,
+          role: user.role ,
+          phone_number: user.phone_number
+        }
```

Comment:
Không cần thiết create trước 1 user, chỉ cần dùng param
```suggestion
    let(:valid_user_params) do
      {
          name: 'valid_name',
          email: 'valid_email@example.com',
          password: 'valid_password',
          password_confirmation: 'valid_password',
          role: :renter ,
          phone_number: '12345678'
      }
    end
    context 'create with valid user' do
      it 'create a new user' do
        expect { post(:create, params: valid_user_params) }.to change(User, :count).by(1)
```